portfolio/src/main/webapp/script.js

```diff
@@ -226,3 +226,43 @@ function requestTranslation() {
   const languageQuery = updateQueryString('language', selectedLanguage);
   refreshComments(languageQuery);
 }
+
+/** Creates a map and adds it to the page. */
+function initMap() {
+  // Try HTML5 geolocation.
+  var map = new google.maps.Map(
+      document.getElementById('map'), {center: {lat: 42.280827, lng: -83.743034}, zoom: 5});
```

Comment:
You might not want to have your exact location broadcast across the internet...maybe just move it to a quad at UMich haha. Or drop off some sig figs.

Reply:
Haha good point, that was not my exact location. I just typed in Ann Arbor and that was something that came up. I don't live there but thanks for the suggestion.